backend/app/logins.py

```diff
@@ -994,12 +994,17 @@ def get_userinfo(login: LoginStatusDep):
 
 @router.post("/refresh-dev-flatpaks")
 def do_refresh_dev_flatpaks(request: Request, login: LoginStatusDep):
-    if login.state == LoginState.LOGGED_OUT:
+    if login.state == LoginState.LOGGED_OUT or login.user is None:
         return Response(status_code=401)
```

Comment:
Might do `raise HTTPException(status_code=401, detail="not_logged_in")` to provide a little more detail. Same below.

Reply:
Updated